dockstore-client/pom.xml

```diff
@@ -296,9 +293,21 @@
             <groupId>jakarta.annotation</groupId>
             <artifactId>jakarta.annotation-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-validator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
```

Comment:
I believe the version for jersey stuff is inherited and controlled via property in bom, so you should be able to leave this out as in https://github.com/dockstore/dockstore/blob/develop/dockstore-webservice/pom.xml#L249-L252

Reply:
If I don't use this dependency I get the following error

```
[INFO] --- maven-dependency-plugin:3.3.0:analyze-only (analyze) @ dockstore-client ---
[WARNING] Used undeclared dependencies found:
[WARNING]    org.glassfish.jersey.media:jersey-media-multipart:jar:2.33:compile
[INFO] Add the following to your pom to correct the missing dependencies: 
[INFO] 
<dependency>
  <groupId>org.glassfish.jersey.media</groupId>
  <artifactId>jersey-media-multipart</artifactId>
  <version>2.33</version>
</dependency>
```